e2e-testing/cypress/locators/SurveyConfigurationLocators.js

```diff
@@ -14,6 +14,7 @@ const SurveyConfigurationLocators = {
   postPreview: '[data-qa="post-preview"]',
   postItem: '[data-qa="post-item"]',
   clearBtn: '[data-qa="clear"]',
+  postDate: '[data-qa="date"]',
```

Comment:
it may be a good idea to rename the `data-qa="date"` to something better relatable like `"post-date"`

Reply:
Thanks for the recommendation on it.